### DISCUSSION TITLE
How to hide a input field?

### DISCUSSION THREAD

**Discussion by lilyntalmers** [ivan-dalmet/formiz#54] [Q&A] [answered]
How to make a Formiz Input Field hidden?

`type="hidden"`

Example: `<input type="hidden" id="fname" name>`

**Comment by ivan-dalmet** [comment, marked as answer]
Hello @lilyntalmers 
You can create a FieldHidden like this :)

## Field

```jsx
const FieldHidden = (props) => {
  const {
    id,
    value,
  } = useField(props)

  return (
      <input
        id={id}
        type="hidden"
        value={value || ''}
      />
  );
}
```

## Usage

```jsx
<FieldHidden name="my-hidden-value" defaultValue="Something" />
```

**Reply by lilyntalmers** [reply]
Hey @ivan-dalmet, I am facing a small issue.

My Input
`<FieldHidden name="_next" defaultValue="https://www.youtube.com/" />`

My Output
`<input id="formiz-form-sxdjz95tt-field-_next" type="hidden" value="https://www.youtube.com/">`

My Expected Output
`<input id="formiz-form-sxdjz95tt-field-_next" type="hidden" value="https://www.youtube.com/"      name="_next">`

Thanks a lot for the help.

**Reply by ivan-dalmet** [reply]
Just update the field to render the name 😊
```jsx
const​ ​FieldHidden​ ​=​ ​(​props​)​ ​=>​ ​{​
  ​const​ ​{​
    id​,​
    value​,​
  ​}​ ​=​ ​useField​(​props​)​

  ​return​ ​(​
      ​<​input​
        ​id​=​{​id​}​
        ​type​=​"hidden"​
        ​value​=​{​value​ ​||​ ​''​}​
        name={props.name} // Update this line
      ​/​>​
  ​)​;​
​}
```

**Reply by lilyntalmers** [reply]
**Thanks @ivan-dalmet,** it worked 😊
